keycloak/keycloak-admin-ui
# Realm settings(keys): add providers sub-tab

## Motivation

https://marvelapp.com/prototype/99i86j9/screen/75627938

## Verification Steps


1. Go to Realm settings section in the old console.
2. Go to Providers -> Add keystore 
3. Add a new keystore and verify it shows in the providers tab.

## Checklist:


## Additional Notes
